ci/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:18.04
+
+RUN ["apt-get", "update"]
+RUN ["apt-get", "install", "-y", "gcc"]
+RUN ["apt-get", "install", "-y", "g++"]
+RUN ["apt-get", "install", "-y", "cmake"]
+
+WORKDIR /build/
+COPY ./ ./project
+
+VOLUME /build/tools
+ENV TOOLS_DIR "/build/tools"
```

Comment:
I would change this to force the user of the container to set the `TOOLS_DIR` environment variable using `--env` and not declare specific volumes within the Dockerfile.

At the usage-location, this will mean that the user has to specify:
```
--env TOOLS_DIR=/build/tools
--volume <something>:/build/tools
```

This will allow the user to change the bash script which relies on the presence of e.g. tools, but dynamically change where they are without rebaking the container.

Reply:
Makes sense, removed them.